src/main/java/pages/LoginPage.java

```diff
@@ -0,0 +1,45 @@
+package pages;
```

Comment:
out of scope

Reply:
the mentioned class [LoginPage.java] was not added from my side.